tests/main.cpp

```diff
@@ -1,2 +1,30 @@
-#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
-#include "doctest.h"
\ No newline at end of file
+#define DOCTEST_CONFIG_NO_UNPREFIXED_OPTIONS
+#define DOCTEST_CONFIG_IMPLEMENT
+#include "doctest.h"
+#include "Luau/Common.h"
+
+LUAU_FASTFLAG(LuauSolverV2)
+LUAU_DYNAMIC_FASTINT(LuauTypeSolverRelease)
+
+int main(int argc, const char** argv)
+{
+    for (int i = 1; i < argc; ++i)
+    {
+        if (argv[i][0] != '-')
+            continue;
+        if (strcmp(argv[i], "--new-solver") == 0)
+        {
+            FFlag::LuauSolverV2.value = true;
+            DFInt::LuauTypeSolverRelease.value = 645;
```

Comment:
can we use max int here so that we don't have to keep bumping it and it will auto work for all future releases?